### PR TITLE
grants admin message && visible toggle

### PR DESCRIPTION
**Pull request by owocki**
##### Description

add sthe ability for an admin to show a message to visitors of a grant
Screen Shot 2021-03-05 at 4 47 43 PM

and also the ability to toggle whethere a grant is even visible on the site at all.

##### Refers/Fixes

convo today on gathertown

##### Testing

tested locally

### CODE REVIEW

File: app/grants/views.py [hunk 1]
```diff
@@ -1339,6 +1339,8 @@ def grant_details(request, grant_id, grant_slug):
                 pk=grant_id
             )
 
+        if not grant.visible:
+            raise Grant.DoesNotExist
```
Comment (thelostone-mc) [inline app/grants/views.py:1343]: ^ I'm not very clear on why we have this here @owocki

**Comment by octavioamu** [comment]
need to be from/to #8500 as many things changed on that areas

File: app/grants/views.py (hunk 1 above)
Comment (owocki) [reply app/grants/views.py]: updated to Http404

**Comment by owocki** [comment]
> need to be from/to #8500 as many things changed on that areas

is already merged :/

**Comment by gdixon** [comment]
Style changes:

Screenshot 2021-03-16 at 21 12 34

Let me know if there is any "snagging" here and I can make those changes 🙌

**Review by gdixon — APPROVED** [review]

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**